src/reference/interactive/index.pug

```diff
@@ -86,62 +88,67 @@ block reference
+    
+    p Developers can use buttons to enable participants to vote, cause in-game actions to happen, or control in-game entities.
 
-    p Buttons have a number of properties which can be edited in both the Interactive Studio and from a Game Client.
+    p Buttons are customizable. These button properties can be edited from both the Interactive Studio and a Game Client:
     ol(style='list-style-type: decimal')
-        li Displayed text
-        li Spark Cost
-        li Progress Bar Width - Displayed at the bottom of a button.
-        li Disabled State - Disabled buttons cannot be interacted with.
-        li Cooldown Duration - Prevents interaction until it expires.
+        li Display text string
+        li Spark cost (For more info about sparks, go to #[a(href='#what-are-sparks') What are Sparks?]
+        li Width of the progress bar, which is displayed at the bottom of a button
+        li Disable property - Disabled buttons cannot be interacted with
+        li Cooldown duration - Prevents interaction until it expires
 
     h5#what-are-sparks What are Sparks?
 
-    p Sparks are Beam's virtual currency. Participants earn them while watching or participating in streams everywhere on Beam.
+    p Sparks are Beam's virtual currency. Participants earn them while watching or participating in live broadcasts everywhere on Beam.
 
     h5#spark-transactions Spark Transactions
     .figure
         img(src='/img/reference/interactive/TransactionLifecycle.svg' alt='Diagram of a transaction\'s life cycle')
     p.caption Diagram of a spark transaction's life cycle, showing its transition betwen states.
 
-    p When a button with a spark cost is pressed, it creates a transaction. To deduct sparks from a participant a #[a(href='#the-game-client') Game Client] must #[code capture] the transaction. If a transaction remains uncaptured for 5 minutes, it expires, and the cost associated with the transaction is not deducted from the user's spark balance.
+    p When a button with a spark cost is pressed, it creates a transaction. To deduct sparks from a participant, a #[a(href='#the-game-client') Game Client] must #[code capture] the transaction. If a transaction remains uncaptured for 5 minutes, it automatically expires, and the cost associated with the transaction is not deducted from the participant's spark balance.
 
-    p This mechanism allows the #[a(href='#the-gameclient') Game Client] to decide if a button push should deduct sparks from a participant. This feature is a great way to ensure that input from a participant is used for something in the experience before deducting sparks from their balance.
+    p This mechanism allows the #[a(href='#the-gameclient') Game Client] to decide whether sparks should be deducted from a participant. This feature is a great way to ensure that input from a participant has been converted into the expected action associated with the button click before deducting sparks from their balance.
```

Comment:
using "click" here is insensitive to our other input methods. would "activation" work?